apps/src/aiTutor/views/AITutorFooter.tsx

```diff
@@ -17,21 +18,42 @@ interface AITutorFooterProps {
   renderAITutor: boolean;
 }
 
+// Helper function to extract active file contents from a MultiFileSource
+const getActiveFileContents = (multiFileSource: MultiFileSource): string => {
```

Comment:
I think you could reuse this function instead: https://github.com/code-dot-org/code-dot-org/blob/a5a216382964b26dfd7a6995a7620344d2e88150/apps/src/lab2/projects/utils.ts#L125